daemon/response.go

```diff
@@ -79,6 +79,10 @@ type Meta struct {
 	Change            string   `json:"change,omitempty"`
 }
 
+func newMeta(_ interface{}) *Meta {
```

Comment:
Why using an interface{} here instead of *state.State ?

Reply:
because when I add tests for this I don't want to have to build a whole `*state.State`, but at the same time I want the delta to be small (check the follow-up to see what I mean)